sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/filters/TargetingFilter.java

```diff
@@ -39,12 +39,12 @@ public class TargetingFilter implements FeatureFilter {
     /**
      * users field in the filter
      */
-    protected static final String USERS = "users";
+    protected static final String USERS = "Users";
```

Comment:
We're using `users` before, if we change to `Users`, will it break some curstomers's application? For example, one customer had a targeting filter and using `users` with lower case. The application may not work when upgrade latest version with uppper case?

Reply:
So, I have corrected these to be the correct values of `Users`/`Groups`, but now I've added a second call with the lowercase version just in case.